docs/v3/core/v3.0.rst

```diff
@@ -634,13 +634,12 @@ mandatory names:
 ``codecs``
 ^^^^^^^^^^
 
-    Specifies a list of codecs to be used for encoding and decoding chunks. The
-    value must be an array of objects, each object containing a member with
-    ``name`` whose value is a string referring to a v3 codec specification. The
-    codec object may also contain a ``configuration`` object which consists of
-    the parameter names and values as defined by the corresponding codec
-    specification.  Since an ``array -> bytes`` codec must be specified, the
-    list cannot be empty.
+    Specifies a list of codecs to be used for encoding and decoding chunks.  The
+    value MUST be an array of objects, where each object contains a member with the name
+    ``name`` whose value is a string. The ``name`` member identifies the specification for the codec.  A
```

Comment:
Previously the requirement was `name` refers to a v3 codec spec, so I guess there would be a catalogue of v3 codecs where I could go an look up the specification.

Now the name is abitrary, so given the name how do I know where to find the codec specification, so if I get a bit of zarr data I can find out how to decode it.

And how do we avoid naming conflicts between community codecs? Or is this not our concern?

Reply:
I think my concerns are resolved by changes under "Extension codecs" below, so maybe worth either repeating a bit of that or signposting that section here?